tests/scene/test_path_follow_3d.h

```diff
@@ -50,7 +50,7 @@ TEST_CASE("[PathFollow3D] Sampling with progress ratio") {
 	path->add_child(path_follow_3d);
 
 	path_follow_3d->set_progress_ratio(0);
-	CHECK(path_follow_3d->get_transform().get_origin().is_equal_approx(Vector3(0, 0, 0));
+	CHECK(path_follow_3d->get_transform().get_origin().is_equal_approx(Vector3(0, 0, 0)); 
```

Comment:
```suggestion
	CHECK(path_follow_3d->get_transform().get_origin().is_equal_approx(Vector3(0, 0, 0));
```
You somehow added a random space here